src/Workspaces/Core/Portable/Workspace/WorkspaceChangeEventArgs.cs

```diff
@@ -21,21 +23,47 @@ public class WorkspaceChangeEventArgs : EventArgs
         public WorkspaceChangeKind Kind { get; }
 
         /// <remarks>
-        /// If linked documents are being changed, there may be multiple events with the same
-        /// <see cref="OldSolution"/> and <see cref="NewSolution"/>.
+        /// If linked documents are being changed, there may be multiple events with the same <see cref="OldSolution"/>
+        /// and <see cref="NewSolution"/>.  Will never be <see langword="null"/>, even for a <see
```

Comment:
>Will never be <see langword="null"/> [](start = 44, length = 36)

Seems redundant to the annotation (or lack thereof). Would be better to say something like "Note that the workspace starts with its solution set to an empty solution. SolutionAdded replaces the previous solution, which might be the empty one."

Reply:
sure.